mne/io/tag.py

```diff
@@ -546,3 +522,28 @@ def _int_item(x):
 
 def _float_item(x):
     return float(x.item())
+
+
+def _matrix_info(tag):
+    matrix_coding = tag.type & 0xFFFF0000
+    if matrix_coding == 0 or tag.size == 0:
+        return None
+    matrix_type = tag.type & 0x0000FFFF
+    matrix_coding_dict = {
+        FIFF.FIFFT_MATRIX: "dense",
+        FIFF.FIFFT_MATRIX | FIFF.FIFFT_SPARSE_CCS_MATRIX: "sparse CCS",
+        FIFF.FIFFT_MATRIX | FIFF.FIFFT_SPARSE_RCS_MATRIX: "sparse RCS",
+    }
+    _check_option("matrix_coding", matrix_coding, list(matrix_coding_dict.keys()))
```

Comment:
[nitpick, feel free to reject]
```suggestion
    _check_option("matrix_coding", matrix_coding, list(matrix_coding_dict))
```